examples/irsa/main.tf

```diff
@@ -70,13 +70,7 @@ module "eks" {
       name                 = "worker-group-1"
       instance_type        = "t2.medium"
       asg_desired_capacity = 1
-      tags = [
-        {
-          "key"                 = "k8s.io/cluster-autoscaler/enabled"
-          "propagate_at_launch" = "false"
-          "value"               = "true"
-        }
-      ]
+      autoscaling_enabled  = true
```

Comment:
Why can't we remove completely remove autoscaling management from this module ? We can add in the doc that user should add required tags like https://github.com/terraform-aws-modules/terraform-aws-eks/pull/716/files#diff-a2651bd231ac6e9334ebf2cf3bd8fa76L73-L79

Reply:
I was thinking just leave that option in as it's an easy way to add the required tags without having to look up what they are. But yeah I guess it's cleaner to just remove this as well.

Reply:
OK I've removed it also.